code/game/machinery/computer/card.dm

```diff
@@ -50,42 +50,31 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 
 	light_color = LIGHT_COLOR_BLUE
 
+/obj/machinery/computer/card/examine(mob/user)
+	. = ..()
+	if(scan || modify)
+		. += "<span class='notice'>Alt-click to eject the ID card.</span>"

```

Comment:
you're going to get two "alt click to eject" messages because you're calling the parent that also has this.